src/app/pages/SettingPage.tsx

```diff
@@ -138,7 +138,7 @@ function SettingPage() {
           {userConfig.chatgptMode === ChatGPTMode.API ? (
             <div className="flex flex-col gap-2">
               <div className="flex flex-col gap-1">
-                <p className="font-medium text-sm">API Key</p>
+                <p className="font-medium text-sm">API {t('Key')}</p>
```

Comment:
I don't think this need translation, so does Host, Model

Reply:
ok，I will revert it